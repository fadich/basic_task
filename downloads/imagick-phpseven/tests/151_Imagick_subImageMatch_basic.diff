001+ Similarity is out of range.
002+ float(0.57447962985124)
001- Similarity in range
004- x : 50
005- y : 50
005+ x : 55
006+ y : 47