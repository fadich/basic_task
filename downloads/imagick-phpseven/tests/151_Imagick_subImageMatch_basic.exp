Similarity in range
height : 25
width : 25
x : 50
y : 50
Ok