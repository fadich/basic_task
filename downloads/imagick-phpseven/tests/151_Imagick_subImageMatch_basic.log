
---- EXPECTED OUTPUT
Similarity in range
height : 25
width : 25
x : 50
y : 50
Ok
---- ACTUAL OUTPUT
Similarity is out of range.
float(0.57447962985124)
height : 25
width : 25
x : 55
y : 47
Ok
---- FAILED
