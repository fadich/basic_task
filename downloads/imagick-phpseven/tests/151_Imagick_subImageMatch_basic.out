Similarity is out of range.
float(0.57447962985124)
height : 25
width : 25
x : 55
y : 47
Ok